src/WebJobs.Script/WebJobs.Script.csproj

```diff
@@ -46,7 +46,7 @@
     <PackageReference Include="Microsoft.ApplicationInsights.PerfCounterCollector" Version="2.20.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer" Version="2.20.0" />
     <PackageReference Include="Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel" Version="2.20.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.34-11957" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.36-dev638198518306956085" />
```

Comment:
Will update this version once webjobs is released - https://github.com/Azure/azure-webjobs-sdk/pull/2981